README.md

```diff
@@ -197,50 +197,64 @@ Currently [**g++10**](https://askubuntu.com/questions/1192955/how-to-install-g-1
 
 A step-by-step tutorial on how to create, compile, and link a new C++ Project with `jluna` can be found [here](./docs/installation.md). It is recommended that you follow this guide closely, instead of trying to resolve issues on your own.
 
-### For Advanced Users Only
+### Creating a new Project from Scratch
 
-Users familiar with C++ and cmake can go through the following steps (on unix-based operating systems):
+`jluna` offers a one-line wizard for installing it and creating a new project, along with its own, finished `CMakeLists.txt` and a hello world. 
 
-Install:
+> this feature is only available on unix systems
 
-+ `g++-11` (or `clang++-12`)
-+ `julia 1.7+`
-+ `cmake 3.16+`
+To do so, download `init.sh` [here](https://raw.githubusercontent.com/Clemapfel/jluna/cmake_rework/install/init.sh). 
 
-Then execute (in the same directory as your `CMakeLists.txt`):
+Then execute (in the same folder you downloaded `init.sh` to):
 
-```bash
-git clone https://github.com/Clemapfel/jluna.git
+```cpp
+/bin/bash init.sh <Project Name> <Projects Path> [<C++ Compiler> = clang++-12]
+```
+Where
++ `<Project Name>` is the name of your desired project folder, for example `MyProject`
++ `<Projects Path>` is the root path to your new project folder, for example `/home/user/Desktop`
++ `<C++ Compiler>` is one of `g++-10`, `g++-11`, `clang++-12`
 
-export JULIA_PATH=$(julia -e "println(joinpath(Sys.BINDIR, \"..\"))")
+The bash script will create a folder in `<Project Path>/<Project Name>` (i.e. `/home/user/Desktop/MyProject`), clone jluna, build it, then, create for you a full project with a working `hello world`. `init.sh` even compiles the new project once. to make sure everything works.
 
-mkdir build
-cd build
-cmake -D CMAKE_CXX_COMPILER=g++-11 .. # or clang++-12
-make
+If errors appear at any point, head to [troubleshooting](./docs/installation.md#troubleshooting).
 
-./JLUNA_TEST
+### Installing `jluna` Globally
 
-cd ..
-rm -r build
-```
+Execute, in any public directory
 
-Where `JULIA_PATH` needs to be set at the time of compilation.
+```bash
+git clone https://github.com/Clemapfel/jluna
+cd jluna
+mkdir build
+cd build
+cmake .. -DCMAKE_CXX_COMPILER=<C++ Compiler> -DCMAKE_INSTALL_PREFIX=<Install Path>
+make install
+```
 
-Link against `jluna/libjluna.so`, `jluna/libjluna_c_adapter.so` and `$ENV{JULIA_PATH}/lib/libjulia.so`.
+Where 
++ `<C++ Compiler>` is one of `g++-10`, `g++-11`, `clang++-12`
++ `<Install Path>` is the install directory, for example `usr/local`
 
-Add `"${CMAKE_SOURCE_DIR}/jluna"` and `"$ENV{JULIA_PATH}/include/julia"` to your include directories.
+Afterwards, you can make `jluna` available to your library using 
 
-Then you can make `jluna` available to your library using:
+```cmake
+find_library(jluna REQUIRED
+    NAMES libjluna.so libjluna.dll.a libjluna.dll libjluna.lib
+    PATHS <Install Path>)
 
-```cpp
-#include <julia.h>
-#include <jluna.hpp>
+target_link_libraries(<Your Library> ${jluna} ${<Julia>})
```

Comment:
These instructions have me raising my eyebrows continuously. Please revise this document using this example: https://github.com/friendlyanon/cmake-init-shared-static/blob/master/BUILDING.md#install

I'm also not sure what the goal of `init.sh` is either.